subprojects/core/src/main/java/org/gradle/api/internal/tasks/SnapshotTaskInputsBuildOperationResult.java

```diff
@@ -114,8 +146,25 @@ public String getPropertyName() {
         }
 
         @Override
+        public Set<String> getPropertyAttributes() {
+            InputFilePropertySpec propertySpec = propertySpec(propertyName);
+            return ImmutableSortedSet.<String>naturalOrder()
+                .add(FilePropertyAttribute.fromNormalizerClass(propertySpec.getNormalizer()).name())
+                .add(FilePropertyAttribute.from(propertySpec.getDirectorySensitivity()).name())
+                .add(FilePropertyAttribute.from(propertySpec.getLineEndingNormalization()).name())
+                .build();
```

Comment:
```suggestion
            return ImmutableSortedSet.of(
                FilePropertyAttribute.fromNormalizerClass(propertySpec.getNormalizer()).name(),
                FilePropertyAttribute.from(propertySpec.getDirectorySensitivity()).name(),
                FilePropertyAttribute.from(propertySpec.getLineEndingNormalization()).name()
            );
```

This can be simplified.

Reply:
agreed and done